pkgs/games/openra/builds.nix

```diff
@@ -0,0 +1,34 @@
+pkgs:
+
+let
+  f = import ./common.nix;
+  common = f (builtins.intersectAttrs (builtins.functionArgs f) pkgs // {
+    inherit (pkgs) stdenv;
+    lua = pkgs.lua5_1;
+    inherit (pkgs.gnome3) zenity;
```

Comment:
You don't need zenity to run the game?

Reply:
No, but it is nice to have a prompt show up when an error occurs rather than having to check the journal or running it from the command line to figure out why it is not running.